sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLViewSuite.scala

```diff
@@ -37,11 +37,21 @@ class SQLViewSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     sqlContext.sql(s"DROP TABLE IF EXISTS jt")
   }
 
-  test("nested views") {
-    withView("jtv1", "jtv2") {
+  test("nested views (interleaved with temporary views)") {
+    withView("jtv1", "jtv2", "jtv3", "temp_jtv1", "temp_jtv2", "temp_jtv3") {
       sql("CREATE VIEW jtv1 AS SELECT * FROM jt WHERE id > 3").collect()
       sql("CREATE VIEW jtv2 AS SELECT * FROM jtv1 WHERE id < 6").collect()
       checkAnswer(sql("select count(*) FROM jtv2"), Row(2))
+
+      // Checks temporary views
+      sql("CREATE TEMPORARY VIEW temp_jtv1 AS SELECT * FROM jt WHERE id > 3").collect()
```

Comment:
Nit: For DDL commands like this one, you don't need to call `.collect()`. Unlike `SELECT` queries, commands are executed eagerly.